build/CMakeFiles/feature_tests.c

```diff
@@ -0,0 +1,37 @@
+
+const char features[] = {"\n"
+                         "C_FEATURE:"
+                         #if (__GNUC__ * 100 + __GNUC_MINOR__) >= 304
+                         "1"
+                         #else
+                         "0"
+                         #endif
+                         "c_function_prototypes\n"
+                         "C_FEATURE:"
+                         #if (__GNUC__ * 100 + __GNUC_MINOR__) >= 304 && defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
+                         "1"
+                         #else
+                         "0"
+                         #endif
+                         "c_restrict\n"
+                         "C_FEATURE:"
+                         #if (__GNUC__ * 100 + __GNUC_MINOR__) >= 406 && defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201000L
+                         "1"
+                         #else
+                         "0"
+                         #endif
+                         "c_static_assert\n"
+                         "C_FEATURE:"
+                         #if (__GNUC__ * 100 + __GNUC_MINOR__) >= 304 && defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
+                         "1"
+                         #else
+                         "0"
+                         #endif
+                         "c_variadic_macros\n"
+
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Redundant blank line at the end of a code block should be deleted.  [whitespace/blank_line] [3]